docs/cli/index.html

```diff
@@ -496,6 +496,13 @@
     publish
   </a>
   
+</li>
```

Comment:
The docs files get built automatically and shouldn't be manually changed (they are generated from the api function docstrings)